Font.js

```diff
@@ -132,8 +131,14 @@ class Font extends EventManager {
      */
     defineFontFace(name, url, options) {
         if (typeof document !== "undefined") {
-            let format = getFontCSSFormat(url);
-            if (!format) return;
+            let format;
+
+            try {
+                format = getFontCSSFormat(url);
+            } catch(_e) {
+                return;
+            }
+
```

Comment:
Rather than try/catching, this can be an `if (!(format = getFontCSSFormat(url)) return;` as that would assign-and-propagate `undefined` from the `getFontCSSFormat` exiting without a return value after dispatching the error event.

Reply:
See my other comment. If we'd let `getFontCSSFormat` return void and only propagate errors via the EventManager, error handling in `loadFont` becomes more complex.

With this code, if the font type cannot be determined this will happen:

1. From `loadFont`, `fromDataBuffer` is called which in turn calls `getFontCSSFormat`.
2. `getFontCSSFormat` will throw, which is caught by `fetch(...).catch(...)` and turned into an Event which is dispatched.

What do you think? If you do still want to make `getFontCSSFormat` return void on error and dispatch the error directly from `getFontCSSFormat` let me know, I could try to make the change and see how it turns out.